README.md

```diff
@@ -154,9 +154,9 @@ References
 
 8. Gazebo breakout discussions (2:00 pm, 20 mins)
 
-9. Break (2:20 pm, 10 minutes)
+9. Stretch break (2:20 pm, 10 minutes)
```

Comment:
Maybe this isn't fair, now that we made the ROS 2 breakout 30 minutes. Should Gazebo breakout also be 30 minutes, and then push back the stretch break to 2:30 pm, and everything subsequently, to end at 4:50 pm?

Reply:
I think as long as we end by 5, that is good.  So yeah, I'd advocate for extending the break.

Reply:
Okay. Done in 161d0c5
We'll have to enforce the time.